apps/site-projects/src/pages/[slug].jsx

```diff
@@ -29,66 +22,60 @@ export const getStaticPaths = async () => {
 
 export const getStaticProps = async (context) => {
   const { slug } = context.params;
-  const { data: project } = await axios.get(
-    `${env().STRAPI_URL}/projects/${slug}`,
-    {
-      headers: {
-        Accept: "application/json, text/plain, */*",
-        "User-Agent":
-          "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.107 Safari/537.36",
-      },
-    }
-  );
+  console.log(slug);
+  const project = await agent.Projects.list({populate: '*', slug: slug});
 
-  if (!project) {
-    return {
-      notFound: true,
-    };
-  }
+  // if (!project) {
+  //   return {
+  //     notFound: true,
+  //   };
```

Comment:
Could we just delete this altogether instead of commenting?

Reply:
removed

Reply:
@mfvil I think this has to be uncommented instead